playbooks/pmm2-client_integration.yaml

```diff
@@ -0,0 +1,149 @@
+---
+# This playbook does following:
+#   enables Percona testing repository
+#   installs latest version of PMM2 client
+#   does some tests
+
+- hosts: all
+  become: true
+  become_method: sudo
+  vars:
+    pmm_server_address: "@@PMM_SERVER_IP@@"
+
+  tasks:
+  - name: include tasks for test env setup
+    include_tasks: ../tasks/test_prep.yml
+
+  - name: include tasks for enabling test repo
+    include_tasks: ../tasks/enable_testing_repo.yml
+    when: lookup('env', 'install_repo') == "testing" or lookup('env', 'install_repo') == ""
+
+  - name: include tasks for enabling experimental repo
+    include_tasks: ../tasks/enable_experimental_repo.yml
+    when: lookup('env', 'install_repo') == "experimental"
+
+  - name: include tasks for enabling main repo
+    include_tasks: ../tasks/enable_main_repo.yml
+    when: lookup('env', 'install_repo') == "main"
+
+  - name: install PMM client new deb packages
+    apt: name={{item}} update_cache=yes state=latest
+    with_items:
+    - pmm2-client
+    when: ansible_os_family == "Debian"
+
+  - name: install PMM client new rpm packages
+    yum: name={{item}} state=latest
+    with_items:
+    - pmm2-client
+    when: ansible_os_family == "RedHat"
+
+  - name: check that PMM client version is correct
+    command: /package-testing/version_check.sh pmm2
+
+  - name: configure pmm client to connect to server
+    shell: pmm-agent setup --server-address={{ pmm_server_address }}:443 --server-insecure-tls --server-username=admin --server-password=admin $(ip addr show|grep "inet 10."|grep -o "10.*/"|sed 's:/::')
+
+### start PS
+  - name: install Percona Server new deb packages
+    apt: name={{item}} update_cache=no state=latest
+    with_items:
+    - percona-server-server-5.7
+    - percona-server-test-5.7
+    - percona-server-5.7-dbg
+    - percona-server-source-5.7
+    - percona-server-tokudb-5.7
+    when: ansible_os_family == "Debian"
+
+  - name: install Percona Server new rpm packages
+    yum: name={{item}} state=latest
+    with_items:
+    - Percona-Server-server-57
+    - Percona-Server-test-57
+    - Percona-Server-57-debuginfo
+    - Percona-Server-devel-57
+    - Percona-Server-tokudb-57
+    - Percona-Server-shared-57
+    - Percona-Server-client-57
+    when: ansible_os_family == "RedHat"
```

Comment:
You can probably just include these tasks from here: https://github.com/Percona-QA/package-testing/blob/master/tasks/install_ps57.yml
like it was done here: https://github.com/Percona-QA/package-testing/blob/master/playbooks/common_57.yml#L49-L50